test/test.js

```diff
@@ -1143,6 +1143,35 @@ describe('docx', () => {
       .should.not.be.rejectedWith(/src parameter to be set/)
   })
 
+  it('broken image from url is skipped', async () => {
+    const url = 'https://some-server.com/broken-image.png'
+
+    nock('https://some-server.com')
+      .get('/broken-image.png')
+      .replyWithFile(200, path.join(__dirname, 'broken-image.png'), {
+        'content-type': 'image/png'
+      })
+
+    const result = await reporter
+      .render({
+        template: {
+          engine: 'handlebars',
+          recipe: 'docx',
+          docx: {
+            templateAsset: {
+              content: fs.readFileSync(path.join(__dirname, 'image.docx'))
+            }
+          }
+        },
+        data: {
+          src: url
+        }
+      })
+      .should.not.be.rejectedWith(RangeError)

```

Comment:
please make sure that also other errors don't occur. now that we don't check for `RangeError` explicitly I think we can remove the argument in the assertion.